index.js

```diff
@@ -41,26 +41,22 @@ module.exports = function (options) {
     // https://github.com/pgherveou/gulp-awspublish/blob/master/lib/log-reporter.js
     var state;
 
-    if (!file.s3) return cb(null, file);
-    if (!file.s3.state) return cb(null, file);
+    if (!file.s3) return callback(null, file);
+    if (!file.s3.state) return callback(null, file);
     if (options.states &&
-        options.states.indexOf(file.s3.state) === -1) return cb(null, file);
+        options.states.indexOf(file.s3.state) === -1) return callback(null, file);
 
     switch (file.s3.state) {
       case 'update':
-        files.push(file.s3.path);
-        break;
       case 'create':
-        files.push(file.s3.path);
-        break;
```

Comment:
Because in javascript, you can group cases using the following syntax:

```js
switch (foo) {
  case 1:
  case 2:
  case 3:
    bar();
    break;
}
```

It basically avoid the repetition of `files.push(file.s3.path);` 3 times.